Packaging/RedHat/functions

```diff
@@ -46,7 +46,7 @@ prepare_environment()
                 if [ $? -eq 1 ]; then BUILD_TOOLS+=" centos-release-scl-rh"; fi
             fi
             if [ $OS_VERSION == "8" ];then
-                yum config-manager --set-enabled PowerTools
+                yum config-manager --set-enabled powertools
                 rpm -q epel-release 2>&1 > /dev/null
                 if [ $? -eq 1 ]; then BUILD_TOOLS+=" epel-release"; fi
                 rpm -q dnf-plugins-core 2>&1 > /dev/null
```

Comment:
Here we could add additional check for Stream branch. In CentOS it's called "PowerTools", in CentOS Stream - "powertools".

Reply:
It is correct, I skipped that point because to build libdispach 5.3 the cmake greater than 3.15 is necessary and in centos 8 there is only 3.11, I also saw that you made some changes in your project to adapt it to clang 11 which is not in the base either only in Stream 8.

Reply:
OK, I'll fix it.